.env.example

```diff
@@ -0,0 +1,68 @@
+# ┌─────────────────────────┐
+# │         PUBLIC          │
+# └─────────────────────────┘
+
+# tests
+TEST_POSTGRES_DATABASE=test
+TEST_POSTGRES_USERNAME=test
+TEST_POSTGRES_PASSWORD=test
+TEST_POSTGRES_HOST=localhost
+TEST_POSTGRES_PORT=4001
+
+# development
+DEVELOPMENT_POSTGRES_DATABASE=web
+DEVELOPMENT_POSTGRES_USERNAME=web
+DEVELOPMENT_POSTGRES_PASSWORD=web
+DEVELOPMENT_POSTGRES_HOST=localhost
+DEVELOPMENT_POSTGRES_PORT=4000
+
+# api server url
+NEXT_PUBLIC_BACKEND=http://localhost:8080/trpc
+
+# ┌─────────────────────────┐
+# │        DATABASE         │
+# └─────────────────────────┘
+
+# production
+PRODUCTION_POSTGRES_DATABASE=
+PRODUCTION_POSTGRES_USERNAME=
+PRODUCTION_POSTGRES_PASSWORD=
+PRODUCTION_POSTGRES_HOST=
+PRODUCTION_POSTGRES_PORT=
+
+# ┌─────────────────────────┐
+# │    LOCAL DEVELOPMENT    │
+# └─────────────────────────┘ http://localhost:3000
+
+# AUTHENTICATION PROVIDERS
+# these id-secret pairs are required to log in
+# using each authentication provider.
+
+# next-auth (required for authentication providers to work)
+NEXTAUTH_SECRET=hPBF3xnQSHHYrmkiT2cP5TViBr4OcjqXgnAPjwAn
```

Comment:
This is intended.
It's a fresh random 40-length string that's different from the real one.